README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
Please go over this file...
1. Add screenshots.
2. Add a meaningful explanation of what it does and how.
3. add instructions on how to launch all of its parts

Reply:
@simalda